_includes/manuals/2.4/3.6_upgrade.md

```diff
@@ -156,41 +208,47 @@ the installer in noop mode so you can see what would be changed.
 
 To see what would happen
 
-    foreman-installer --noop --dont-save-answers --verbose
+    foreman-installer --noop --verbose
 
 You may see ERRORS such as `/Stage[main]/Foreman_proxy::Register/Foreman_smartproxy[foreman-hostname.domain]:` `Could not evaluate: Connection refused - connect(2)` due to httpd / apache2 service being stopped.  These can be safely ignored.
 
 To apply these changes, run the installer again without options
 
     foreman-installer
 
-#### Step 4 - Restart
-
-Restart the application server (e.g. mongrel, thin, passenger).
-
-On RPM installations, run:
-
-    systemctl start httpd foreman.service foreman.socket
-
-And on Debian/Ubuntu installations, run:
-
-    systemctl start apache foreman.service foreman.socket
-
-#### Optional Step 5 - Reclaim database space
-
-After database migrations, some space can sometimes be reclaimed. It's a good idea to perform a *full* database vacuum for PostgreSQL rather than relying on the autovacuum feature to claim maximum space possible. For PostgreSQL do the following:
-
-    foreman-maintain service stop --exclude postgresql
-    su - postgres -c 'vacuumdb --full --dbname=foreman'
-    foreman-maintain service start
-
-On platforms where foreman-maintain is not available, stop all Foreman-related services except PostgreSQL.
-
-#### Optional Step 6 - Cleanup of RPMs and repositories
-
-On EL7, consider removing unused SCL packages:
-
-    yum erase rh-ruby22\* rh-ruby24\* rh-ror42\* tfm-ror52\*
+##### Optional Step 3 (D) - Update config files
+<div class="upgrade_os upgrade_os_none">
+  <i>*No operating system selected.*</i>
+</div>
+<div class="upgrade_os upgrade_os_el7">
+In order to catch all configuration changes and manage them properly you should install and run
+rpmconf from the EPEL repository along with vim-enhanced (for vimdiff).
+
+{% highlight bash %}
+rpmconf -a --frontend=vimdiff
+{% endhighlight %}
+</div>
+<div class="upgrade_os upgrade_os_el8 upgrade_os_debian10 upgrade_os_ubuntu1804">
+This step is irrelevant for the chosen operating system.
+</div>
```

Comment:
Perhaps only show this whole section only for upgrade_os_el7?

Reply:
I didn't want to write that complex code. If you use HTML tags, you can't use markdown. So `#####` needs to be some HTML tag.